test/system/irohad_test.cpp

```diff
@@ -154,9 +170,114 @@ class IrohadTest : public AcceptanceFixture {
 
   std::string setDefaultParams() {
     return params(
-        config_copy_, path_genesis_.string(), path_keypair_.string(), {});
+        config_copy_, path_genesis_.string(), path_keypair_node_.string(), {});
   }
 
+  void prepareTestData() {
+    keys_manager_admin_.createKeys();
+    keys_manager_node_.createKeys();
+    keys_manager_testuser_.createKeys();
+
+    auto admin_keys = keys_manager_admin_.loadKeys();
+    ASSERT_TRUE(admin_keys);
+
+    auto node0_keys = keys_manager_node_.loadKeys();
+    ASSERT_TRUE(node0_keys);
+
+    auto user_keys = keys_manager_testuser_.loadKeys();
+    ASSERT_TRUE(user_keys);
+
+    shared_model::interface::RolePermissionSet admin_perms{
```

Comment:
Can these definitions be moved to `test/framework/common_constants.hpp|cpp` ?

Reply:
Not sure, because such genesis block is used only in this test. Integration test framework uses all permission for admin, and none for users by default.

Reply:
I defined these specific permissions in order to match (as close as possible) the original genesis block file. I could move these to common constants, either as they are now or even in a simplified form (if some permissions are not necessary for the test). 

I also used some string literals for the genesis block creation. For example: the `"money_creator"` and `"admin"` roles, as well as the `"user"` account (all of these from the original file). 

There is an admin role defined in common constants, but its value is `"admin_role"`, not `"admin"`, so I used the string literal for that. There is a user definition, which I didn't notice before, so I'll fix that. I can also add the money_creator role as well.